ui/desktop/src/components/ApiKeyWarning.tsx

```diff
@@ -1,42 +1,103 @@
 import React from 'react';
 import { Card } from './ui/card';
 import { Bird } from './ui/icons';
+import { ChevronDown } from 'lucide-react';
 
 interface ApiKeyWarningProps {
   className?: string;
 }
 
+interface CollapsibleProps {
+  title: string;
+  children: React.ReactNode;
+  defaultOpen?: boolean;
+}
+
+function Collapsible({ title, children, defaultOpen = false }: CollapsibleProps) {
```

Comment:
This is 🥇, the only thing to add if possible is a copy to clipboard button in the top right to make it easier to copy paste the block